<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HeartWave Simulator</string>
  </property>
  <widget class="QWidget" name="rootWidget">
   <widget class="QGroupBox" name="historyGroupBox">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>11</y>
      <width>200</width>
      <height>579</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>History</string>
    </property>
    <widget class="QListWidget" name="sessionHistoryListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>550</y>
       <width>181</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="historyToolButtonsHBoxLayout">
      <item>
       <layout class="QHBoxLayout" name="historyToolButtonsLeftHBoxLayout">
        <item>
         <widget class="QToolButton" name="deleteSession">
          <property name="toolTip">
           <string>Delete</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../res/icons.qrc">
            <normaloff>:/icons/fontawesome/solid/minus.svg</normaloff>:/icons/fontawesome/solid/minus.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="historyToolButtonsSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="clearSessionHistory">
        <property name="toolTip">
         <string>Clear All</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/fontawesome/solid/trash.svg</normaloff>:/icons/fontawesome/solid/trash.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="sessionGroupBox">
    <property name="geometry">
     <rect>
      <x>228</x>
      <y>12</y>
      <width>561</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>Session</string>
    </property>
    <widget class="BatteryIndicatorWidget" name="batteryIndicator">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Battery Level</string>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>400</y>
       <width>130</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="sessionToolButtonsLayout">
      <item>
       <widget class="QToolButton" name="startSession">
        <property name="toolTip">
         <string>Start</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/fontawesome/solid/play.svg</normaloff>:/icons/fontawesome/solid/play.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="stopSession">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Stop</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/fontawesome/solid/stop.svg</normaloff>:/icons/fontawesome/solid/stop.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="powerButton">
        <property name="toolTip">
         <string>Power</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons.qrc">
          <normaloff>:/icons/fontawesome/solid/power-off.svg</normaloff>:/icons/fontawesome/solid/power-off.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="BreathPacerWidget" name="breathPacer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>521</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
    <widget class="HRVGraphWidget" name="hrvGraph" native="true">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>69</y>
       <width>531</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="CoherenceIndicatorWidget" name="coherenceIndicator">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="interruptSensor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Interrupt Sensor</string>
     </property>
    </widget>
    <widget class="QLabel" name="HRContactLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>42</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>100</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../res/icons.qrc">:/icons/fontawesome/solid/heart-pulse.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <zorder>HRContactLabel</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>batteryIndicator</zorder>
    <zorder>breathPacer</zorder>
    <zorder>hrvGraph</zorder>
    <zorder>coherenceIndicator</zorder>
    <zorder>interruptSensor</zorder>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>450</y>
      <width>561</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="settingsAndMetricsHBoxLayout">
     <item>
      <widget class="QGroupBox" name="metricsGroupBox">
       <property name="title">
        <string>Session Metrics</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>261</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="metricsVBoxLayout">
         <item>
          <layout class="QHBoxLayout" name="sessionLengthHBoxLayout">
           <item>
            <widget class="QLabel" name="sessionLengthLabel">
             <property name="text">
              <string>Session Length (seconds)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="sessionLengthLCDNumber">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="smallDecimalPoint">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="coherenceScoreHBoxLayout">
           <item>
            <widget class="QLabel" name="coherenceScoreLabel">
             <property name="text">
              <string>Coherence Score</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="coherenceScoreLCDNumber">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="smallDecimalPoint">
              <bool>true</bool>
             </property>
             <property name="digitCount">
              <number>5</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="achievementScoreHBoxLayout">
           <item>
            <widget class="QLabel" name="achievementScoreLabel">
             <property name="text">
              <string>Achievement Score</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="achievementScoreLCDNumber">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="smallDecimalPoint">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="coherenceMetricsGroup">
       <property name="title">
        <string>Coherence Metrics</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>262</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="settingsVBoxLayout">
         <item>
          <layout class="QHBoxLayout" name="challengeLevelHBoxLayout">
           <item>
            <widget class="QLabel" name="label1">
             <property name="text">
              <string>High Coherence (%)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="highCoherencePercent">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="smallDecimalPoint">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="breathPacerIntervalHBoxLayout">
           <item>
            <widget class="QLabel" name="label2">
             <property name="text">
              <string>Medium Coherence (%)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="mediumCoherencePercent">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="smallDecimalPoint">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="simulatedCoherenceLevelHBoxLayout">
           <item>
            <widget class="QLabel" name="label3">
             <property name="text">
              <string>Low Coherence (%)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lowCoherencePercent">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="smallDecimalPoint">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BreathPacerWidget</class>
   <extends>QProgressBar</extends>
   <header>../../src/ui/BreathPacerWidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>HRVGraphWidget</class>
   <extends>QWidget</extends>
   <header>../../src/ui/HRVGraphWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CoherenceIndicatorWidget</class>
   <extends>QFrame</extends>
   <header>../../src/ui/CoherenceIndicatorWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BatteryIndicatorWidget</class>
   <extends>QProgressBar</extends>
   <header>../../src/ui/BatteryIndicatorWidget.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sessionHistoryListWidget</tabstop>
  <tabstop>deleteSession</tabstop>
  <tabstop>clearSessionHistory</tabstop>
  <tabstop>startSession</tabstop>
  <tabstop>stopSession</tabstop>
  <tabstop>powerButton</tabstop>
 </tabstops>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
